app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments
+  has_many :likes
+
+  def three_most_recent_posts
+    Post.where(author: self).order(created_at: :desc).limit(3)
+  end
```

Comment:
[OPTIONAL]
- [ ] It's great that you're working with the data effectively! :clap: I'd like to suggest a more efficient approach for retrieving the last three recent posts associated with a user.

  The code `Post.where(author: self).order(created_at: :desc).limit(3)` certainly works, but it can be further simplified. By using `posts.order(created_at: :desc).limit(3)`, you take advantage of the association between the post and its user, which is generally more performant.